build_tools/test_libs.sh

```diff
@@ -19,7 +19,7 @@ fi
 set e  # Ignore errors until artifacts are collected.
 EXIT_CODE=0
 for TARGET in bitstring_test channels_cirq_test circuit_qsim_parser_test expect_test \
-              fuser_basic_test gates_qsim_test hybrid_test matrix_test qtrajectory_test \
+              fuser_basic_test gates_qsim_test hybrid_avx_test matrix_test qtrajectory_avx_test \
```

Comment:
Why did these change?

Reply:
because there is no `hybrid_test` and `qtrajectory_test`...?

Reply:
Huh. Wonder how long it's been like that...